crates/polars-time/src/windows/duration.rs

```diff
@@ -689,7 +691,7 @@ impl Duration {
         }
 
         if d.weeks > 0 {
-            let t_weeks = nsecs_to_unit(self.weeks * NS_WEEK);
+            let t_weeks = nsecs_to_unit(NS_WEEK) * self.weeks;
```

Comment:
this could be cleaner with, say, `MS_WEEK` and `US_WEEK`...but this simple change at least fixes the issue